python/paddle/fluid/tests/unittests/test_matmul_v2_op.py

```diff
@@ -495,6 +495,26 @@ def test_dygraph_fp16(self):
                     y = paddle.to_tensor(input_y)
                     result = paddle.matmul(x, y)
 
+    def test_compute_type_fp32(self):
+        if core.is_compiled_with_cuda():
+            place = core.CUDAPlace(0)
```

Comment:
Does it need to add another test case that will be not finite if set `FLAGS_gemm_use_half_precision_compute_type=True`

Reply:
done